metron-platform/metron-pcap/src/main/java/org/apache/metron/pcap/finalizer/PcapRestFinalizer.java

```diff
@@ -16,27 +16,21 @@
  * limitations under the License.
  */
 
-package org.apache.metron.pcap;
+package org.apache.metron.pcap.finalizer;
 
-import java.util.List;
+import java.util.Map;
 import org.apache.hadoop.fs.Path;
-import org.apache.metron.job.Pageable;
+import org.apache.metron.pcap.config.PcapOptions;
 
-public class PcapFiles implements Pageable<Path> {
-
-  private List<Path> files;
-
-  public PcapFiles(List<Path> files) {
-    this.files = files;
-  }
+/**
+ * Write to HDFS.
+ */
+public class PcapRestFinalizer extends PcapFinalizer {
 
   @Override
-  public Iterable<Path> asIterable() {
-    return files;
+  protected String getOutputFileName(Map<String, Object> config, int partition) {
+    Path finalOutputPath = PcapOptions.FINAL_OUTPUT_PATH.getTransformed(config, Path.class);
+    return String.format("%s/page-%s", finalOutputPath, partition);
```

Comment:
This constant string should be be a `static final String` with a comment about the file format in English.